vpr/src/base/stats.cpp

```diff
@@ -377,8 +377,7 @@ void print_wirelen_prob_dist() {
                 VTR_LOG("Realloc'ing to increase 2-pin wirelen prob distribution array.\n");
                 incr = index - prob_dist_size + 2;
                 prob_dist_size += incr;
-                prob_dist = (float*)vtr::realloc(prob_dist,
-                                                 prob_dist_size * sizeof(float));
+                prob_dist = new float[prob_dist_size];
```

Comment:
Same comment as above.  If realloc is used repeatedly, you could make a utility function to handle the new, copy, delete sequence.

Reply:
Or just make it a vector (probably best of all).